38/js/templateData.js

```diff
@@ -19,6 +19,6 @@ limitations under the License.
 var context={
     "version": "38",
     "dotVersion": "3.8",
-    "fullDotVersion": "3.8.0",
+    "fullDotVersion": "3.8.1-SNAPSHOT",
```

Comment:
Is this correct? Should be `3.8.1` only?

Reply:
I think it should be 3.8.1 yes. I'll fix it shortly

Reply:
fixed